http2_general/test_h2_headers.py

```diff
@@ -223,6 +223,36 @@ def test(self):
         CurlTestBase.run_test(self)
 
 
+class BackendSetCoookieH2(CurlTestBase):
+    """The test checks the correctness of forwarding long headers with
+    duplication in mixed order: put header B between two headers A
```

Comment:
I believe this comment is not relates to current test. As for me, duplication in mixed order should looks like:
```
add_header x-extra-data1 "q";
add_header x-extra-data2 "q";
add_header x-extra-data1 "q";
```

Reply:
Please update copyright year.

Reply:
Comment coems from other test - fixed